docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -81,6 +68,7 @@ we are not provided).
 
 
 ## Lag and Backfill
-Google does not update the search data daily, but has an uncertain update
-frequency. The delay can range from 1 day to 10 days or even more. We check for
+Google does not currently update the search data daily, but usually twice a week.
+Each update will usually the coverage to within three days of the day of the update.
```

Comment:
might be missing a word here: "Each update will usually the coverage". maybe "extend the coverage"?

Reply:
bring or extend would be great